src/hotspot/share/utilities/nativeCallStack.cpp

```diff
@@ -28,6 +28,14 @@
 #include "utilities/globalDefinitions.hpp"
 #include "utilities/nativeCallStack.hpp"
 
+static unsigned calculate_hash(address stack[NMT_TrackingStackDepth]) {
```

Comment:
I know it doesn't change anything semantically, but I'd like to see the int type specifier.

Reply:
Sure, I'll change it before pushing.